### ISSUE TITLE
Broken input with Codeception/WebDriver "fillField()"

### ISSUE THREAD

**Issue by mbrodala** [nathanreyes/v-calendar#1279]
We have quite a few acceptance tests run by CodeCeption/Webdriver which, among others, fill date picker fields managed by v-calendar. This did work fine using `fillField()` up until [v3.0.0-alpha.8](https://github.com/nathanreyes/v-calendar/releases/tag/v3.0.0-alpha.8). However, with [v3.0.0](https://github.com/nathanreyes/v-calendar/releases/tag/v3.0.0) this started to fail.

Debugging the test runs which fortunately are recorded as video we saw that filling the field this way leads to broken input: previously the existing date value from the input field was overwritten as desired. Now e.g. entering `23.03.2023` leads to a value like `NaN.NaN.0NaN`.  (We use `locale: 'de'` for the datepicker, thus the date format is `DD.MM.YYYY`.)

After [some investigation](https://sqa.stackexchange.com/questions/39488/trouble-deleting-nan-in-input-field) we found that `fillField()` internally first uses `clear()` to empty the field and then inputs the desired value via `sendKeys()`. Evidently after `clear()` some processing/validation is going on leading to the broken input.

[The solution](https://stackoverflow.com/a/3254152/6812729) which did work for us was using [`pressKey()`](https://codeception.com/docs/modules/WebDriver#pressKey) to first select the existing value using `Ctrl`+`A` followed by the actual input which then overwrites the previous value:

```php
$I->pressKey(
    // Use selector for date picker field
    ...,
    // Select existing text
    ['ctrl', 'a'],
    // Overwrite with new value
    ...,
);
```

Not sure if this is something v-calendar can or should fix but we wanted to be sure this issue is mentioned somewhere with a solution to be found by others.

**Comment by nathanreyes** [comment]
Yes it is related to #1270 and #1283 and is going into the next patch.

**Comment by nathanreyes** [comment]
This should be fixed in 3.0.2. Feel free to reopen otherwise.

**Comment by mbrodala** [comment]
Thanks, works great and we could revert our workaround. :+1:
